code/modules/reqs/supplypacks.dm

```diff
@@ -507,6 +507,29 @@ EXPLOSIVES
 /datum/supply_packs/explosives/detpack
 	name = "detpack explosives"
 	contains = list(
+		/obj/item/detpack,
+		/obj/item/detpack,
+		/obj/item/detpack,
+		/obj/item/detpack,
+		/obj/item/detpack,
+		/obj/item/detpack,
+		/obj/item/assembly/signaler,
+	)
+	cost = 10
+
+/datum/supply_packs/explosives/detpack
```

Comment:
this is still a duplicate declaration.
look 13 lines above, at line 507, you can see the declaration there is the exact same as the one you typed. This means that the second declaration (the one at line 520) will be the only one to exist and the first one (line 507) wont matter anymore